external/dist/webpack.js

```diff
@@ -16,6 +16,11 @@
 
 var pdfjs = require('./build/pdf.js');
 var PdfjsWorker = require('worker-loader!./build/pdf.worker.js');
-pdfjs.PDFJS.workerPort = new PdfjsWorker();
+
+if ( typeof window !== 'undefined' && 'Worker' in window ) {
```

Comment:
spaces after `(` and before `)` needs to be removed. please also amend/squash commits.